packages/amplify-cli/src/extensions/amplify-helpers/delete-project.ts

```diff
@@ -43,16 +42,44 @@ export const deleteProject = async (context: $TSContext): Promise<void> => {
           printer.warn('Amplify App cannot be deleted, other environments still linked to Application');
         }
       }
+
+      // delete paramater store parameters for each environment
+      const CloudFormationProviderName = 'awscloudformation';
+      const deleteParametersFromParameterStoreFn: (
```

Comment:
```suggestion
      const deleteParametersFromParameterStore: (
```
avoid hungarian-esque notation